### ISSUE TITLE
Youtube has an issue with landscape mode for the mobile site

### ISSUE THREAD

**Issue by xt2345** [kiwibrowser/src.next#542]
![Screenshot_20220316-133816_Kiwi Browser](https://user-images.githubusercontent.com/97930353/158653369-d4fbf312-259e-45bc-8711-8a0d36b8460d.jpg)
for some reason this happens everytime i switch to landscape mode and im not sure on how to fix it.

**Comment by xt2345** [comment]
update: so the issue i found out is that the browser doesnt automatically zoom in to reisze the video to enable scrolling in landscape mode, 
![Screenshot_20220316-160808_Kiwi Browser](https://user-images.githubusercontent.com/97930353/158681513-0e4bf88d-a32a-49a8-b70a-f2fb6c6b983c.jpg)
would it be possible to change the video player to this size in a update?

**Comment by karl-pae** [comment]
Duplicate of #256. Should be closed.

**Comment by xt2345** [comment]
im on the current github version and this still happening.

**Comment by karl-pae** [comment]
Yes. But this is duplicate issue. You can follow issue no 256 for the updates.  Hence it should be closed. 🙂

**Comment by xt2345** [comment]
Tbh im leaving this open unless the bug us taken care of, issue 256 does not say its fixed  so im leaving this open until an update fixes the bug.

**Comment by Ibuprophen** [comment]
@xt2345 , I believe that @karl-pae meant that this is a duplicate of https://github.com/kiwibrowser/src.next/issues/256 and had suggested that this issue be closed to carry the conversation over there. 

Please correct me if I'm wrong... 👍 

I believe that multiple issues open regarding the same thing can potentially be counter productive, but this is just my opinion as I'm not a developer of this repository. 


~Ibuprophen

**Comment by xt2345** [comment]
> @xt2345 , I believe that @karl-pae meant that this is a duplicate of #256 and had suggested that this issue be closed to carry the conversation over there.
> 
> Please correct me if I'm wrong... 👍
> 
> I believe that multiple issues open regarding the same thing can potentially be counter productive, but this is just my opinion as I'm not a developer of this repository.
> 
> ~Ibuprophen

people being ok with bugs is why the quality of software has been on the decline, im using yandex until the bugs get fixed.
